files/image_config/copp/copp_cfg_chassis.j2

```diff
@@ -0,0 +1,111 @@
+{
+    "COPP_GROUP": {
+	    "default": {
+		    "queue": "0",
+		    "meter_type":"packets",
+		    "mode":"sr_tcm",
+		    "cir":"600",
+		    "cbs":"600",
+		    "red_action":"drop"
+	    },
+	    "queue7_group1": {
+		    "trap_action":"trap",
+		    "trap_priority":"4",
+		    "queue": "7"
+	    },
+	    "queue7_group2": {
+		    "trap_action":"copy",
+		    "trap_priority":"4",
```

Comment:
So here we are keeping the trap_priority as 4 itself, and queue as 7, any relation between queue number and trap priority?